packages/react/src/forms/FormControl/FormControl.stories.tsx

```diff
@@ -334,3 +334,25 @@ export const ErrorValidation = () => {
   )
 }
 ErrorValidation.storyName = 'w/ Error Validation - Playground'
+
+export const ErrorValidationWithCheckbox = () => {
+  return (
+    <FormControl validationStatus="error">
+      <FormControl.Label>Day</FormControl.Label>
+      <Checkbox name="day" value="" />
+      <FormControl.Validation>Please select at least one day</FormControl.Validation>
+    </FormControl>
+  )
+}
+ErrorValidationWithCheckbox.storyName = 'w/ Error Validation with Checkbox - Playground'
+
+export const ErrorValidationWithRadio = () => {
+  return (
+    <FormControl validationStatus="error">
+      <FormControl.Label>Day</FormControl.Label>
+      <Radio name="day" value="" />
+      <FormControl.Validation>Please select at least one day</FormControl.Validation>
+    </FormControl>
+  )
+}
+ErrorValidationWithRadio.storyName = 'w/ Error Validation with Radio - Playground'
```

Comment:
As the JSX is almost identical to the existing `CheckboxPlayground` story, it might be helpful to reuse it with different arguments. This should save a few lines, and keep all checkbox stories consistent.

E.g. 

```jsx 
export const ErrorValidationWithCheckbox = args => <CheckboxPlayground {...args} />
ErrorValidationWithCheckbox.storyName = 'w/ Error Validation with Checkbox - Playground'
ErrorValidationWithCheckbox.args = {
  validationStatus: 'error',
}
```

Reply:
Good suggestion - I'll update that!